pkg/sdk/config.go

```diff
@@ -69,6 +69,9 @@ func MergeConfig(baseConfig *gosnowflake.Config, mergeConfig *gosnowflake.Config
 	if baseConfig.Host == "" {
 		baseConfig.Host = mergeConfig.Host
 	}
+	if baseConfig.Warehouse == "" {
```

Comment:
it is a behavior change and as @sfc-gh-jcieslak tells me you have reverted  some changes introduced because of some local setup problems. So I guess it can be reverted too, right?